substrate/frame/assets-freezer/README.md

```diff
@@ -0,0 +1,29 @@
+# Assets Freezer Pallet
```

Comment:
Let's remove the readme, and just have the docs in the rustdoc. Otherwise need to maintain the doc in two places.